resources/grafana/downstream/grafana-dashboard-dataplane-cluster-view.yaml

```
@@ -2286,7 +2288,8 @@ data:
                 "mode": "absolute",
                 "steps": [
                   {
-                    "color": "green"
+                    "color": "green",
```

Can you revert all unnecessary changes like this one? It makes the PR harder to review.
```suggestion
                    "color": "green",
```

---

@lgarciaaco Done!